src/call.cpp

```diff
@@ -2821,7 +2821,9 @@ bool call::process_incoming(const char* msg, const struct sockaddr_storage* src)
 #ifdef RTP_STREAM
     /* Check if message has a SDP in it; and extract media information. */
     if (!strcmp(get_header_content(msg, "Content-Type:"), "application/sdp") &&
-            hasMedia == 1 && !curmsg->ignoresdp) {
+        hasMedia == 1 && !curmsg->ignoresdp) {
+      const char *ptr = get_header_content(msg, "content-length:");
+      if (ptr && atoll(ptr) > 0)
```

Comment:
Please add curly braces, even for single line ifs, like the https://github.com/SIPp/sipp/#contributing readme says.